src/validation/permissionsValidation.js

```diff
@@ -0,0 +1,69 @@
+import { validationResult } from 'express-validator';
+import models from '../models';
+import errorAssignment from '../utils/errorAssignment';
+
+const { roles } = models;
+
+const validateRole = async (request) => {
+  let errors = {};
+  const { affectedRole } = request.body;
+  if (!affectedRole) errors = { ...errorAssignment('Role input is compulsory', 'affectedRole') };
+  if (!/^\d+$/.test(affectedRole)) errors = { ...errorAssignment('Role input must be a number', 'affectedRole') };
+
+  if (!errors.affectedRole && affectedRole) {
+    const userRole = await roles.findOne({
+      where: { id: affectedRole },
+      raw: true
+    });
+    if (!userRole) {
+      errors = { ...errorAssignment('Check role input and try again', 'affectedRole') };
+    } else {
+      request.userRole = userRole.id;
+    }
+  }
+  return errors;
+};
+const validatePermissionField = async (request) => {
+  let errors = {};
+  let { addPermission, removePermission } = request.body;
+  addPermission = addPermission ? addPermission.trim() : null;
+  removePermission = removePermission ? removePermission.trim() : null;
+
+  // eslint-disable-next-line
```

Comment:
is there a reason we are disabling the newline here?

Reply:
The line length is longer than 110 characters as per eslint check. I cannot seem to break the code into a second line so I had to disable the rule

Reply:
has this been resolved?